src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
 import {update} from 'app/actionCreators/projects';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 
+class PlatformList extends React.Component {
```

Comment:
Should we move this into its own module?

Reply:
👍